### DISCUSSION TITLE
Should only the owners submit their package?

### DISCUSSION THREAD

**Discussion by SmartManoj** [microsoft/winget-pkgs#3682] [General] [answered]
` Have you checked that there aren't other open pull requests for the same manifest update/change?`
Found In the PR template.

**Comment by OfficialEsco** [comment, marked as answer]
No. That would make the Update Process VERY slow.
The good thing about WinGet is that anyone can update a manifest to the latest version.

**Comment by SmartManoj** [comment]
Difference Between making a PR for a package and  suggest manifest in issue
